frontend/src/components/landing/Reviews.module.scss

```diff
@@ -0,0 +1,294 @@
+@import "../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.wrapper {
+  display: grid;
+  justify-items: center;
+
+  .reviews {
+    display: flex;
+    flex-direction: column;
+    max-width: $content-max;
+    gap: $spacing-lg;
+    width: 100%;
+    margin: 0 auto;
+    align-items: center;
+    padding: $spacing-md $spacing-lg 0 $spacing-lg;
+    justify-content: center;
+
+    @media screen and #{$mq-lg} {
+      gap: $layout-md;
+      flex-direction: row;
+    }
+  }
+}
+
+.left-container {
+  display: flex;
+  flex-direction: row;
+  gap: $spacing-xl;
+}
+
+.right-container {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-2xl;
+}
+.logo-container {
+  display: flex;
+  font-family: $font-stack-base;
+  color: $color-white;
+  width: $layout-2xl;
+  flex-direction: column;
+  align-items: center;
+  border-radius: $border-radius-lg;
+  padding: $spacing-md 0;
+  background-color: $color-purple-30;
+
+  .logo {
+    width: $layout-xl;
+    padding: $spacing-sm;
+  }
+
+  .logo-title {
+    @include text-body-md;
+    font-weight: 400;
+  }
+
+  .logo-text {
+    @include text-title-xs;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+}
+
+.rating-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  gap: $spacing-xs;
+
+  .rating {
+    .title {
+      @include text-title-2xl;
+      font-family: $font-stack-firefox;
+      font-weight: 600;
+    }
+
+    .text {
+      @include text-body-sm;
+      color: $color-grey-40;
+      font-family: $font-stack-base;
+      font-weight: 500;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.star {
+  color: $color-yellow-50;
+}
+
+.empty-star {
+  color: $color-white;
+  stroke: $color-grey-20;
+  stroke-width: 2px;
+}
+
+.chevron {
+  appearance: none;
+  border: none;
+  padding: 0;
+  background-color: $color-white;
+  color: $color-grey-30;
+  width: $layout-sm;
+  cursor: pointer;
+}
+
+.quotation-icon {
+  display: flex;
+  position: absolute;
+
+  @media screen and #{$mq-md} {
+    left: 0;
+  }
+
+  svg {
+    color: $color-purple-60;
+    height: $layout-sm;
+  }
+}
+
+.reviews-container {
+  display: flex;
+  overflow-x: hidden;
+  flex-direction: row;
+  gap: $spacing-lg;
+  // magic number: set to avoid shifting - no character limit on reviews yet.
+  min-height: calc($layout-2xl * 2);
+  position: relative;
+
+  @media screen and #{$mq-xl} {
+    // magic number: set to avoid shifting - no character limit on reviews yet.
+    min-height: 210px;
+  }
+
+  .review-container {
+    display: flex;
+    align-items: center;
+
+    @media screen and #{$mq-xl} {
+      align-items: flex-start;
+    }
+  }
+
+  .review {
```

Comment:
Looking at the design, isn't this element supposed to have a white background? (And the left- and right-hand side buttons rounded corners?)

Reply:
white background removed from buttons - reviewed by Eduardo and Tony. My screen contrast was set to 100% so I couldn't see the background before.